<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_Pw</class>
 <widget class="QDialog" name="change_Pw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="change_Btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(252, 233, 79), stop:1 rgba(196, 160, 0));
border: 2px solid rgb(255, 215, 0);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string>비밀번호 변경</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="now_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ch_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ck_ch_pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: white;
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>변경할 비밀번호 확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>변경할 비밀번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>현재 비밀번호</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
